### DISCUSSION TITLE
Passkeys

### DISCUSSION THREAD

**Discussion by misterakko** [RobThree/TwoFactorAuth#113] [Ideas]
I was wondering if there's any intention to add to the band the new promising kid on the block, Passkeys. If so, I'm available to help.

**Comment by NicolasCARPi** [comment]
https://www.eff.org/deeplinks/2023/10/what-passkey

**Comment by willpower232** [comment]
I would say that webauthn is a completely different specification and probably worth using a separate library rather than twisting this one to fit.

A quick look at https://github.com/herrjemand/awesome-webauthn suggests https://webauthn.spomky-labs.com/ and https://github.com/web-auth/webauthn-framework/tree/4.8.x/src/webauthn looks like a thorough library for this purpose.

**Comment by RobThree** [comment]
Agree. No intention to add it to this library. Sorry.
